farmdata2_modules/fd2_tabs/fd2_example/fd2_example.info

```diff
@@ -5,8 +5,4 @@ package = FarmData2
 
 stylesheets[all][] = '../resources/fd2.css'
 scripts[] = '../resources/FarmData2.js'
-scripts[] = '../resources/CustomTableComponent.js'
-scripts[] = '../resources/DateSelectionComponent.js'
-scripts[] = '../resources/DateRangeSelectionComponent.js'
-scripts[] = '../resources/DropdownWithAllComponent.js'
 scripts[] = '../resources/FarmOSAPI.js'
```

Comment:
Should these lines have been removed from fd2_example.info?  Won't that break the ex1 page?

Reply:
Weird, I don't have those lines in my local branch. I'll make a new commit and double check that they've went away.